_layouts/default.html

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="shortcut icon" href="/img/favicon.ico" type="image/x-icon">
+    <link rel="icon" href="/img/favicon-32.png" sizes="32x32">
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="/assets/bootstrap/css/bootstrap.min.css">
+    <link rel="stylesheet" href="/assets/bootstrap/css/bootstrap-select.min.css">
+    <!-- icon css-->
+    <link rel="stylesheet" href="/assets/elagent-icon/style.css">
+    <link rel="stylesheet" href="/assets/niceselectpicker/nice-select.css">
+    <link rel="stylesheet" href="/assets/animation/animate.css">
+    <link rel="stylesheet" href="/assets/mcustomscrollbar/jquery.mCustomScrollbar.min.css">
+    <link rel="stylesheet" href="/assets/prism/prism.css">
+    <link rel="stylesheet" href="/css/style-main.css">
+    <link rel="stylesheet" href="/css/responsive.css">
```

Comment:
Added the following after this line
https://github.com/LovelaceAcademy/Learn/blob/310c596f0186ad784d966a9859e7b7d7c47822f5/index.html#L21